style/rails/README.md

```diff
@@ -19,8 +19,15 @@ Rails
 * Use `def self.method`, not the `scope :method` DSL.
 * Use the default `render 'partial'` syntax over `render partial: 'partial'`.
 * Use `link_to` for GET requests, and `button_to` for other HTTP verbs.
+* If you break up an argument list for a macro,
+  keep the first argument on the same line as the macro,
```

Comment:
I'd prefer to see non-hash arguments on one line until that gets crazy, then hash arguments on their own.

```
create :user, :admin,
  name: "Caleb Thompson",
  address: "..."

fill_form :user, :update,
  name: "Caleb"

extract_options foo: :bar, baz: :bat
```